--- a/tests/KDEInstallDirsTest/relative_or_absolute/CMakeLists.txt
+++ b/tests/KDEInstallDirsTest/relative_or_absolute/CMakeLists.txt
@@ -1,6 +1,7 @@
 project(KDEInstallDirs.relative_or_absolute)
 cmake_minimum_required(VERSION 2.8.12)
 
+set(KDE_INSTALL_USE_QT_SYS_PATHS FALSE)
 set(ECM_KDE_MODULE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../../kde-modules)
 set(ECM_MODULE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../../modules)
 set(CMAKE_MODULE_PATH "${ECM_KDE_MODULE_DIR}")
