Index: extra-cmake-modules-1.0.0/kde-modules/KDEInstallDirs.cmake
===================================================================
--- extra-cmake-modules-1.0.0.orig/kde-modules/KDEInstallDirs.cmake
+++ extra-cmake-modules-1.0.0/kde-modules/KDEInstallDirs.cmake
@@ -323,16 +323,16 @@ _define_relative(CMAKEPACKAGEDIR LIBDIR
     CMAKECONFIG_INSTALL_PREFIX)
 
 # Qt-specific vars
-_define_relative(QTPLUGINDIR LIBDIR "plugins"
+_define_relative(QTPLUGINDIR LIBDIR "qt5/plugins"
     "Qt plugins"
     QT_PLUGIN_INSTALL_DIR)
 _define_relative(PLUGINDIR QTPLUGINDIR ""
     "Plugins"
     PLUGIN_INSTALL_DIR)
-_define_relative(QTQUICKIMPORTSDIR QTPLUGINDIR "imports"
+_define_relative(QTQUICKIMPORTSDIR QTPLUGINDIR "qt5/imports"
     "QtQuick1 imports"
     IMPORTS_INSTALL_DIR)
-_define_relative(QMLDIR LIBDIR "qml"
+_define_relative(QMLDIR LIBDIR "qt5/qml"
     "QtQuick2 imports"
     QML_INSTALL_DIR)
 
@@ -432,7 +432,7 @@ _define_relative(DBUSSYSTEMSERVICEDIR DB
 
 
 
-_define_absolute(SYSCONFDIR "etc"
+_define_absolute(SYSCONFDIR "/etc"
     "read-only single-machine data"
     SYSCONF_INSTALL_DIR)
 _define_relative(CONFDIR SYSCONFDIR "xdg"
Index: extra-cmake-modules-1.0.0/modules/ECMGeneratePriFile.cmake
===================================================================
--- extra-cmake-modules-1.0.0.orig/modules/ECMGeneratePriFile.cmake
+++ extra-cmake-modules-1.0.0/modules/ECMGeneratePriFile.cmake
@@ -82,7 +82,7 @@
 # (To distribute this file outside of extra-cmake-modules, substitute the full
 #  License text for the above reference.)
 
-set(ECM_MKSPECS_INSTALL_DIR mkspecs/modules CACHE PATH "The directory where mkspecs will be installed to.")
+set(ECM_MKSPECS_INSTALL_DIR ${LIB_INSTALL_DIR}/qt5/mkspecs/modules CACHE PATH "The directory where mkspecs will be installed to.")
 
 function(ECM_GENERATE_PRI_FILE)
   set(options )
