Index: extra-cmake-modules-0.0.13/kde-modules/KDEInstallDirs.cmake
===================================================================
--- extra-cmake-modules-0.0.13.orig/kde-modules/KDEInstallDirs.cmake	2014-05-04 20:59:53.000000000 +0100
+++ extra-cmake-modules-0.0.13/kde-modules/KDEInstallDirs.cmake	2014-05-16 10:22:36.000000000 +0100
@@ -206,7 +206,7 @@
 # this exists as a convenience for frameworks
 set(KF5_INCLUDE_INSTALL_DIR          "${INCLUDE_INSTALL_DIR}/KF5")
 
-_set_fancy(QT_PLUGIN_INSTALL_DIR     "${LIB_INSTALL_DIR}/plugins"                  "The install dir where Qt plugins will be installed (default is ${LIB_INSTALL_DIR}/plugins)")
+_set_fancy(QT_PLUGIN_INSTALL_DIR     "${LIB_INSTALL_DIR}/qt5/plugins"                  "The install dir where Qt plugins will be installed (default is ${LIB_INSTALL_DIR}/plugins)")
 _set_fancy(PLUGIN_INSTALL_DIR        "${QT_PLUGIN_INSTALL_DIR}"                    "The install dir where framework and application plugins will be installed (default is ${QT_PLUGIN_INSTALL_DIR})")
 _set_fancy(IMPORTS_INSTALL_DIR       "${QT_PLUGIN_INSTALL_DIR}/imports"            "The install dir where QtQuick1 imports will be installed")
 _set_fancy(QML_INSTALL_DIR           "${LIB_INSTALL_DIR}/qml"                      "The install dir where QtQuick2 imports will be installed")
@@ -229,7 +229,7 @@
 _set_fancy(XDG_DIRECTORY_INSTALL_DIR "${SHARE_INSTALL_PREFIX}/desktop-directories" "The XDG directory")
 _set_fancy(XDG_MIME_INSTALL_DIR      "${SHARE_INSTALL_PREFIX}/mime/packages"       "The install dir for the xdg mimetypes")
 
-_set_fancy(SYSCONF_INSTALL_DIR       "etc"                                         "The sysconfig install dir (default etc)")
+_set_fancy(SYSCONF_INSTALL_DIR       "/etc"                                         "The sysconfig install dir (default etc)")
 _set_fancy(CONFIG_INSTALL_DIR        "${SYSCONF_INSTALL_DIR}/xdg"                  "The config file install dir")
 _set_fancy(AUTOSTART_INSTALL_DIR     "${CONFIG_INSTALL_DIR}/autostart"             "The install dir for autostart files")
 _set_fancy(MAN_INSTALL_DIR           "${SHARE_INSTALL_PREFIX}/man"                 "The man install dir (default ${SHARE_INSTALL_PREFIX}/man)")
Index: extra-cmake-modules-0.0.13/modules/ECMGeneratePriFile.cmake
===================================================================
--- extra-cmake-modules-0.0.13.orig/modules/ECMGeneratePriFile.cmake	2014-05-04 20:59:53.000000000 +0100
+++ extra-cmake-modules-0.0.13/modules/ECMGeneratePriFile.cmake	2014-05-16 10:27:18.258307113 +0100
@@ -82,7 +82,7 @@
 # (To distribute this file outside of extra-cmake-modules, substitute the full
 #  License text for the above reference.)
 
-set(ECM_MKSPECS_INSTALL_DIR mkspecs/modules CACHE PATH "The directory where mkspecs will be installed to.")
+set(ECM_MKSPECS_INSTALL_DIR ${LIB_INSTALL_DIR}/qt5/mkspecs/modules CACHE PATH "The directory where mkspecs will be installed to.")
 
 function(ECM_GENERATE_PRI_FILE)
   set(options )
